.htaccess

```
@@ -144,12 +144,12 @@ DirectoryIndex index.php index.html index.htm
     # Serve gzip compressed CSS files if they exist and the client accepts gzip.
     RewriteCond %{HTTP:Accept-encoding} gzip
     RewriteCond %{REQUEST_FILENAME}\.gz -s
-    RewriteRule ^(.*)\.css $1\.css\.gz [QSA]
+    RewriteRule ^(.*css_[a-zA-Z0-9-_]+)\.css$ $1\.css\.gz [QSA]
```

Since the filename starts with *css_*, the `.*` part is not necessary, or `Rewrite` would handle a `randomcharacterscss_aZ09.css` filename.

---

@kiamlaluno are you suggesting something like this then?:

```suggestion
    RewriteRule ^(.*)/(css_[a-zA-Z0-9-_]+)\.css$ $1/$2\.css\.gz [QSA]
```

---

Since that should be handling values like `/files/css/css_xxGQ2.css`, that would work. I think those files will never be accessed with a URL like `http://example.com/css_file_to_use.css`.